src/models/UserData.ts

```
@@ -20,7 +23,9 @@ export default class UserData {
   }
 
   commitCurrentGame (): void {
-    this.games.push(this.currentGame)
-    this.saveGames()
+    if (this.currentGame !== undefined) {
+      this.games.push(this.currentGame)
+      this.saveGames()
+    }
```

We should probably throw an error, or return a boolean, to indicate whether this succeeded or not.